specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-04-01/managedClusters.json

```diff
@@ -193,6 +193,7 @@
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/accessProfiles/{roleName}/listCredential": {
       "post": {
+        "deprecated": true,
```

Comment:
Please update the description to explain/tell users what they should do instead.

Reply:
Updated. Thanks.

Reply:
Looks like you meant the API description? I thought you meant the PR description. Let me update that. Thanks.

Reply:
Yes, sorry, I meant the API description. Giving developers the information on what they should call instead is tremendously helpful when they have to update their code. I would even consider putting a link out to documentation that shows old and new code.

Reply:
I have updated the description and added links to the recommended APIs that should be used instead. Since they are pretty straightforward and not much different from old API, I wouldn't worry about adding a link to code snippet. Let me know if you feel that's strictly required.